cx.ring.libRingClient.json

```diff
@@ -0,0 +1,82 @@
+{
+    "name": "libRingClient",
+    "buildsystem": "cmake",
+    "config-opts":[
+        "-DCMAKE_BUILD_TYPE=Release"
+    ],
+    "sources":[
+        {
+            "type": "git",
+            "url": "https://gerrit-ring.savoirfairelinux.com/ring-lrc.git",
+            "commit": "a5544675c5fc65c91dc160435bebf6d7de37c769"
+        }
+    ],
+    "modules":[
+        "cx.ring.RingDaemon.json",
+        {
+            "name": "qt5-qtbase",
+            "cleanup-platform": [ "/bin", "/mkspecs" ],
+            "build-options" : {
+                "arch" : {
+                    "x86_64" : {
+                        "config-opts" : [
+                            "-debug", "-reduce-relocations"
+                        ]
+                    },
+                    "arm" : {
+                        "config-opts" : [
+                            "-no-reduce-relocations", "-no-use-gold-linker", "-optimize-size"
+                        ]
+                    }
+                }
+            },
+            "config-opts": [
+                "-prefix /app",
+                "-confirm-license",
+                "-opensource",
+                "-shared",
+                "-platform linux-g++",
+                "-optimized-qmake",
+                "-nomake examples",
+                "-nomake tests",
+                "-system-harfbuzz",
+                "-system-sqlite",
+                "-accessibility",
+                "-dbus-linked",
+                "-fontconfig",
+                "-glib",
+                "-icu",
+                "-openssl-linked",
+                "-no-pch",
+                "-no-rpath",
+                "-no-separate-debug-info",
+                "-reduce-relocations",
```

Comment:
This is different to the [KDE runtime's version](https://github.com/KDE/flatpak-kde-runtime/blob/master/org.kde.Sdk.json#L81) and seems to be causing a [build failure on aarch64](https://flathub.org/builds/#/builders/6/builds/880).